### DISCUSSION TITLE
k8s version not supported in AWS

### DISCUSSION THREAD

**Discussion by jnan1234** [egovernments/Digit-Core#276] [Bugs]
Hi,

While trying to setup DIGIT in aws using DIGIT-DEVOPS(https://github.com/egovernments/DIGIT-DevOps.git) release branch getting deployment errors and observed that AWS EKS not supported k8s version which mentioned in release branch code (please see below 2nd image).

 kindly assist further step and suggest working branch for updated DIGIT code.

![image](https://github.com/egovernments/Digit-Core/assets/41353401/23370df6-2244-490a-92f6-0bb7aba9dbc4)

![image](https://github.com/egovernments/Digit-Core/assets/41353401/53e33e4d-8238-4bcc-ab96-bffa75ac7263)

**Comment by naresh-egov** [comment]
K8s Version 1.22 is no longer supported by AWS, please use the version 1.23 or later versions.

**Reply by naresh-egov** [reply]
Please use the following repo to upgrade your exisitng EKS Cluster.
https://github.com/egovernments/DIGIT-DevOps/tree/release-1.27-kubernetes

Note: Please do rolling update
